docs/quickstart.rst

```diff
@@ -566,20 +567,24 @@ For this example, we'll create a FileSourceConnector, a FileSinkConnector and di
         -e CONNECT_CONFIG_STORAGE_TOPIC="quickstart-config" \
         -e CONNECT_OFFSET_STORAGE_TOPIC="quickstart-offsets" \
         -e CONNECT_STATUS_STORAGE_TOPIC="quickstart-status" \
+        -e CONNECT_CONFIG_STORAGE_REPLICATION_FACTOR=1 \
+        -e CONNECT_OFFSET_STORAGE_REPLICATION_FACTOR=1 \
+        -e CONNECT_STATUS_STORAGE_REPLICATION_FACTOR=1 \
         -e CONNECT_KEY_CONVERTER="org.apache.kafka.connect.json.JsonConverter" \
         -e CONNECT_VALUE_CONVERTER="org.apache.kafka.connect.json.JsonConverter" \
         -e CONNECT_INTERNAL_KEY_CONVERTER="org.apache.kafka.connect.json.JsonConverter" \
         -e CONNECT_INTERNAL_VALUE_CONVERTER="org.apache.kafka.connect.json.JsonConverter" \
         -e CONNECT_REST_ADVERTISED_HOST_NAME="localhost" \
         -e CONNECT_LOG4J_ROOT_LOGLEVEL=DEBUG \
+        -e CONNECT_LOG4J_LOGGERS=org.reflections=ERROR \
         -v /tmp/quickstart/file:/tmp/quickstart \
         confluentinc/cp-kafka-connect:3.3.0-SNAPSHOT
 
   As you can see in the above command, we tell Connect to refer to the three topics we create in the first step of this Connect tutorial. Let's check to make sure that the Connect worker is up by running the following command to search the logs:
```

Comment:
If we're going to have Connect create the internal topics, we should change the first sentence to:

> As you can see in the above command, we tell Connect to use three topics to store configuration, offsets, and status, and it will create these correctly if they don't yet exist. We're only running one broker, so we do need to specify a replication factor of `1` for these topics.